### PR TITLE
Added the Tiberian Sun Temple of Nod

### PR DESCRIPTION
**Pull request by Mailaender**
Completing the tech tree.

### CODE REVIEW

**Comment by pchote** [comment]
Looks fine to me, but this doesn't complete the tech tree.  We are still missing at least the Nod tiberium facility and the GDI firestorm generator and hunter/seeker upgrade plug.

File: mods/ts/rules/nod-infantry.yaml [hunk 1]
```diff
@@ -98,7 +98,7 @@ MHIJACK:
 	Buildable:
 		Queue: Infantry
 		BuildPaletteOrder: 60
-		Prerequisites: ~nahand, natech # natech must be natmpl
+		Prerequisites: ~nahand, natmpl
```
Comment (pchote) [inline mods/ts/rules/nod-infantry.yaml]: There are several other actor types that need `natech` &rarr; `natmpl`.

Comment (pchote) [inline mods/ts/rules/nod-infantry.yaml]: Cyborg Command and hunter-seeker.

File: mods/ts/rules/nod-structures.yaml [hunk 2]
```diff
@@ -277,6 +277,34 @@ NATECH:
...
+		Prerequisites: natech, ~structures.nod
+	ProvidesPrerequisite@buildingname:
+	Valued:
+		Cost: 2000
+	Tooltip:
+		Name: Temple of Nod
+		Description: Provides access to advanced Nod technologies.
+	Building:
+		Footprint: xxxx xxxx xxxx
+		Dimensions: 4,3
```
Comment (pchote) [inline mods/ts/rules/nod-structures.yaml]: Rules.ini specifies Adjacent: 3

Comment (Mailaender) [inline mods/ts/rules/nod-structures.yaml]: https://github.com/LipkeGu/TSIniFiles/blob/master/art.ini#L1155 is the relevant line.

File: mods/ts/rules/nod-infantry.yaml (hunk 1 above)
Comment (Mailaender) [inline mods/ts/rules/nod-infantry.yaml]: That has nothing to do with this line.

Comment (pchote) [inline mods/ts/rules/nod-infantry.yaml]: Those other actor types need the same change that you have done here.

File: mods/ts/rules/nod-structures.yaml (hunk 2 above)
Comment (pchote) [inline mods/ts/rules/nod-structures.yaml]: I was trying to be helpful by commenting these issues at the appropriate trait parts of our yaml.  If you'd prefer I'll make future comments at the PR level and let you hunt through our definitions yourself.

Comment (Mailaender) [inline mods/ts/rules/nod-structures.yaml]: Just try to place them somewhere relevant. Otherwise your unrelated comments will stay there forever and people will never merge it, because there are outstanding remarks.

Comment (pchote) [inline mods/ts/rules/nod-structures.yaml]: The definition belongs on this trait, (at your choice) on the line below the one I have commented on.  I fail to see how that is not relevant.

Comment (Mailaender) [inline mods/ts/rules/nod-structures.yaml]: It was very irritating when you added this without any further context such as "try adding ..., too".

File: mods/ts/rules/nod-infantry.yaml (hunk 1 above)
Comment (Mailaender) [inline mods/ts/rules/nod-infantry.yaml]: I mean you commented this line, I changed the cyborg commando as well, but this won't auto-hide, because the line is unrelated which will probably lead to everyone thinking this has unresolved issues.

**Comment by pchote** [comment]
:+1:

**Comment by GraionDilach** [comment]
:+1:
